src/cnfizers/Cnfizer.cc

```diff
@@ -427,33 +427,6 @@ bool Cnfizer::assertClause (PTRef f)
     throw OsmtInternalException("UNREACHABLE: Unexpected situation in Cnfizer");
 }
 
-//
-// Retrieve the formulae at the top-level.  Ignore duplicates
-//
-void Cnfizer::retrieveTopLevelFormulae (PTRef root, vec<PTRef> &top_level_formulae)
-{
-    vec<PTRef> to_process;
-
-    Map<PTRef, bool, PTRefHash> seen;
-
-    to_process.push (root);
-
-    while (to_process.size() != 0) {
-        PTRef f = to_process.last();
-        to_process.pop();
-        Pterm &cand_t = logic.getPterm (f);
-
-        if (logic.isAnd (f)) {
-            for (int i = cand_t.size() - 1; i >= 0; i--) {
```

Comment:
For some reason this was iterating the conjuncts from the end. This is going to change now, so I should run the experiments to check the effect on run time.

Reply:
Probably some old stuff from that I added to try to reproduce opensmt1 behaviour while hunting performance bugs long time ago.  It's ok to change it.